stdlib/public/core/Algorithm.swift

```diff
@@ -119,17 +121,19 @@ public struct EnumeratedSequence<Base : Sequence> : Sequence {
 public struct EnumerateGenerator<Base : IteratorProtocol> { }
 
 @available(*, unavailable, renamed: "EnumeratedSequence")
-public struct EnumerateSequence<Base : Sequence> {}
+public struct EnumerateSequence<Base : Sequence> { }
 
 extension EnumeratedIterator {
-  @available(*, unavailable, message: "use the 'enumerated()' method on the sequence")
+  // Calling `makeIterator()` on an `EnumeratedSequence` is currently
+  // the only public way to obtain an `EnumeratedIterator`.
+  @available(*, unavailable, message: "Construct an `EnumeratedSequence` and call `makeIterator()` on it.")
```

Comment:
Since you can't construct an `EnumeratedSequence` directly, perhaps this should say `"Call 'enumerated().makeIterator()' on the sequence."`